src/components/FlowRunsFilterGroup.vue

```diff
@@ -1,64 +1,80 @@
+        <DateRangeSelect v-model="range" />
       </p-label>
       <p-label label="States">
-        <StateNameSelect v-model:selected="filter.flowRuns.state.name" empty-message="All run states" multiple />
+        <StateNameSelect v-model:selected="state" empty-message="All run states" multiple />
       </p-label>
     </div>
     <div class="flow-runs-filter-group__row">
       <p-label label="Flows">
-        <FlowCombobox v-model:selected="filter.flows.id" empty-message="All flows" multiple />
+        <FlowCombobox v-model:selected="flow" empty-message="All flows" multiple />
       </p-label>
       <p-label label="Deployments">
-        <DeploymentCombobox v-model:selected="filter.deployments.id" empty-message="All deployments" multiple />
+        <DeploymentCombobox v-model:selected="deployment" empty-message="All deployments" multiple />
       </p-label>
       <p-label label="Work Pools">
-        <WorkPoolCombobox v-model:selected="filter.workPools.name" empty-message="All pools" multiple />
+        <WorkPoolCombobox v-model:selected="workPool" empty-message="All pools" multiple />
       </p-label>
       <p-label label="Tags">
-        <FlowRunTagsInput v-model:selected="filter.flowRuns.tags.name" :filter="filter" empty-message="All tags" />
+        <FlowRunTagsInput v-model:selected="tag" empty-message="All tags" />
       </p-label>
     </div>
     <p-label class="flow-runs-filter-group__search" label="Search">
-      <SearchInput v-model="flowRunNameLike" placeholder="Search by flow run name" label="Search by flow run name" />
+      <SearchInput v-model="flowRunName" placeholder="Search by flow run name" label="Search by flow run name" />
     </p-label>
   </div>
 </template>
 
 <script lang="ts" setup>
-  import { media, PLabel } from '@prefecthq/prefect-design'
-  import { useDebouncedRef } from '@prefecthq/vue-compositions'
-  import { computed, ref } from 'vue'
-  import DateRangeInputWithFlowRunHistory from '@/components/DateRangeInputWithFlowRunHistory.vue'
+  import { PLabel } from '@prefecthq/prefect-design'
+  import { usePatchRef } from '@prefecthq/vue-compositions'
+  import { computed } from 'vue'
+  import DateRangeSelect from '@/components/DateRangeSelect.vue'
   import DeploymentCombobox from '@/components/DeploymentCombobox.vue'
   import FlowCombobox from '@/components/FlowCombobox.vue'
   import FlowRunTagsInput from '@/components/FlowRunTagsInput.vue'
   import SearchInput from '@/components/SearchInput.vue'
   import StateNameSelect from '@/components/StateNameSelect.vue'
   import WorkPoolCombobox from '@/components/WorkPoolCombobox.vue'
-  import { useRecentFlowRunsFilterFromRoute } from '@/compositions/filters'
+  import { SavedSearchFilter } from '@/models/SavedSearch'
 
-  const flowRunNameLike = ref<string>()
-  const flowRunNameLikeDebounced = useDebouncedRef(flowRunNameLike, 1200)
+  const props = defineProps<{
+    filter: SavedSearchFilter,
+    name: string,
+  }>()
```

Comment:
This took me a second to understand - `name` is actually the prop used to filter the list based on flow run name, right? Maybe a more apt prop name would be `nameSearch` or `flowRunNameSearch`. As is, I couldn't figure out why a `FlowRunsFilterGroup` component needed a name

Reply:
Also I'm a little surprised that this isn't captured in the filter as an `ilike` property

Reply:
updated the name. The name search (along with the subflow runs toggle) are not included in saved searches. Which is why the name is a separate v-model here.